### DISCUSSION TITLE
代码块中有双引号该怎么解决

### DISCUSSION THREAD

**Discussion by PurePasserby** [tiddly-gittly/TidGi-Desktop#262] [Q&A]
小白想问一下代码块中出现 ` “ ` 该如何解决
```c++
<<code src:" 
map["Aa"]="bb"
" language:"c++" title:"">>
```
这行代码就会有问题，' map[ '后的代码不会显示，需要通过什么符号转义吗？

类型改成 markdown 后代码块是好的，但不能像这样 [[其他]] 跳转。

tiddly wiki 5 的三个 ```后面不能像markdown一样加代码类型如c++，该如何高亮代码呢？

**Comment by linonetwo** [comment]
还是群里问吧，我写代码也没啥空答疑，群里偶尔回下

ChenYe ***@***.***> 于2022年8月22日周一 14:55写道：

**Reply by PurePasserby** [reply]
好的，能发一下群号吗？我没找到有群号

**Reply by linonetwo** [reply]
看 https://zhuanlan.zhihu.com/p/140473235 这里有各种联系方式

**Reply by PurePasserby** [reply]
好的，谢谢
